proto/frequenz/api/dispatch/dispatch.proto

```diff
@@ -189,15 +194,12 @@ message DispatchUpdateRequest {
   // The component selector
   DispatchComponentSelector selector = 6;
```

Comment:
uh `optional string type = 3;` should be removed. Changing that would create quite some issues.

Reply:
Also keep in mind that start_time has to be in the future! Can you please also add that to the documentation? Only start_time with a future date should be allowed to be updated.